src/Compilers/CSharp/Portable/Symbols/Tuples/TupleTypeSymbol.cs

```diff
@@ -75,6 +91,14 @@ public TupleTypeSymbol(ImmutableArray<Location> locations, NamedTypeSymbol under
             return Create(locationOpt, underlyingType, elementLocations, elementNames);
         }
 
+        public static TupleTypeSymbol Create(NamedTypeSymbol tupleCompatibleType)
+        {
+            return Create(ImmutableArray<Location>.Empty,
+                          tupleCompatibleType, 
+                          default(ImmutableArray<Location>), 
+                          default(ImmutableArray<string>));
+        }
```

Comment:
The location is the location where the tuple type is declared in source or in metadata by using special attribute. This method is not used for these cases. 

---
In reply to: [62241050](https://github.com/dotnet/roslyn/pull/11047#discussion_r62241050) [](ancestors = 62241050)